app/assets/stylesheets/components/_card.scss

```diff
@@ -2,7 +2,11 @@
   background-color: $white;
   max-width: $container-skinny-width;
 
-  &-wide { max-width: 100%; }
+  &-wide {
+    max-width: 100%;
+    padding-left: 0;
+    padding-right: 0;
```

Comment:
Yep! I mentioned that in this comment https://github.com/18F/identity-idp/pull/1336#discussion_r110470135. I think I'd prefer to do the followup PR if that is cool with you, that way I won't have to make changes a second time when I introduce the new layout.